lib/net/modules/tcp.toit

```diff
@@ -197,7 +197,13 @@ tcp-listen_ socket-resource-group address port backlog:
   #primitive.tcp.listen
 
 tcp-write_ socket-resource-group descriptor data from to:
-  #primitive.tcp.write
+  #primitive.tcp.write: | error |
+    if error != "WRONG_BYTES_TYPE": throw error
```

Comment:
Add a comment to explain why this isn't using `io.primitive-redo-chunked-io-data_`.

Reply:
done.